src/zenml/steps/step_environment.py

```diff
@@ -102,10 +105,10 @@ def cache_enabled(self) -> bool:
         return self._cache_enabled
 
     @property
-    def pipeline_requirements(self) -> List[str]:
-        """The name of the currently running step.
+    def docker_configuration(self) -> "DockerConfiguration":
```

Comment:
Is this used anywhere? I am not sure if I am missing anything.

Reply:
Not in my PR, but @safoinme added the `pipeline_requirements` property to the step environment so he could then use it in this custom code deployment PR. Now the pipeline requirements don't exist anymore but got replaced by the docker config.